mappings/net/minecraft/level/biome/Biome.mapping

```diff
@@ -25,24 +25,24 @@ CLASS net/minecraft/class_153 net/minecraft/level/biome/Biome
 	FIELD field_898 biomes [Lnet/minecraft/class_153;
 	METHOD method_785 createBiomeArray ()V
 	METHOD method_786 getBiome (DD)Lnet/minecraft/class_153;
-		ARG 0 x
-		ARG 2 z
-	METHOD method_796 getSkyColour (F)I
-		ARG 1 temperature
+		ARG 0 temperature
+		ARG 1 humidity
 	METHOD method_787 getClimateBiome (FF)Lnet/minecraft/class_153;
 		ARG 0 temperature
-		ARG 1 rainfall
+		ARG 1 humidity
```

Comment:
why

Reply:
The mc wiki calls it rainfall for beta versions

Reply:
humidity describes it better imo: rainfall can vary but humidity stays the same most of the time

Reply:
fine ill change it